cartesian/pfss_cartesian_obj.py

```diff
@@ -0,0 +1,133 @@
+import warnings
+
+import numpy as np
+import sunpy.map
+from astropy.io import fits
+import utils
+
+class Input:
+    r"""
+    Input to PFSS modelling.
+
+    This code uses a Cartesian geometry (x,y,z) for performing the extrapolation. 
+    Thus, make sure the grid defined is cartesian, and that solutions obtained are 
+    valid at all. 
+    Parameters
+    ----------
+    bz : sunpy.map.GenericMap
+        Boundary condition of LOS magnetic field at the inner surface.
+
+    Z_max : float
+        Max height in vertical direction to which extrapolation is performed, in Mm (Mega meters)
+
+    dz : float
+        Grid spacing. 
+    """
+    def __init__(self, bz, Z_max, dz):
+        if not isinstance(bz, sunpy.map.GenericMap):
+            raise ValueError('br must be a SunPy Map')
+        if np.any(~np.isfinite(bz.data)):
+            raise ValueError('At least one value in the input is NaN or '
+                             'non-finite. The input must consist solely of '
+                             'finite values.')
+        if np.mean(bz.data) > 1e-10:
```

Comment:
Can get rid of this check. I just got rid of it for the main PFSS solver, as I think it's fine to assume that the PFSS equations automatically ignore the monopole component.

Reply:
The mean() check only gives a warning. If you think its fine to remove, I will remove it  - but let me know.